### DISCUSSION TITLE
Leesson 7: Deployment failure with `make deploy-sepolia`

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [Cyfrin/foundry-full-course-cu#297] [Q&A] [answered]
Video:  https://www.youtube.com/watch?v=OlxnxWLC4dQ&list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk&index=103 

The error message is as follows: 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/b33c01bf-068f-419e-895b-ccab9fbb15a5)

My attempt:
1. I tried replacing the new `SEPOLIA_RPC_URL`, but it still reports an error!
2. Tried restarting VS Code and re-executing the `make deploy-sepolia` command, but it still gives me an error!
3. Returned to the Makefile & .env files in the instructor's video and found no errors.

**Comment by umairahmed88** [comment]
share your code

**Comment by ShubhSensei** [comment, marked as answer]
share your `.env` file & Makefile

**Comment by IAm0xAxe** [comment]
I'm so sorry, @umairahmed88 @SensationalShubham 
I apologize for not seeing your replies until now.
Regarding my question, I finally realized it was an error in my command!

**Reply by cromewar** [reply]
please share what was your solution so this might be useful for the community. Congrats for solving this by your own, great job!
